src/pages/zh-cn/guides/troubleshooting.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 故障排查
+description: 需要帮助吗？卡在什么地方了？我们已经帮你解决了。
+---
+
+Astro 提供了几个不同的工具来帮助你排除故障和调试你的代码。
+
+## 常见错误信息
+
+下面是你在终端可能看到的一些常见错误信息，它们可能意味着什么，以及如何处理它们。
+
+### 转化失败，出现 X 错误
+
+这条信息经常出现，因为目前 Astro 要求导入和导出语句必须在 `.astro`文件顶层。
+
+**解决方案**：把你的导入和导出语句写在你的组件脚本的顶部。
+
+**状态**：现有限制，正在进行修复。
+
+**不确定这是否是你的问题？**
+检查一下是否有其他人报告过[这个问题](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Transform+failed+with+*+error)!
+
+### 不能在模块外使用导入语句
+
+在 Astro 组件中，`<script> `标签会被挂起并默认作为 [JS 模块](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)加载。如果你在标签中包含了 [`is:inline` 指令](/zh-cn/reference/directives-reference/#isinline)或任何其他属性，将不使用该行为。
+
+**解决方案**：如果你在 `<script>` 标签中添加了任何属性，你也必须添加 `type="module"` 属性，以便能够使用导入语句。
+
+**状态**：预期 Astro 行为。
+
+**不确定这是你的问题？**
+检查一下是否有其他人报告过[这个问题](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Cannot+use+import+statement)!
+
+### 无法渲染组件
+
+这表示你已经导入并在你的Astro模板中使用的组件出现了错误。
+
+#### 常见原因
+
+这可能是由于在渲染时试图访问 `window` 或 `document` 对象造成的。默认情况下，Astro 会[同构](https://en.wikipedia.org/wiki/Isomorphic_JavaScript)渲染你的组件，这意味着它在浏览器运行时间不可用的服务器上运行。你可以使用 [`client:only` 指令](/zh-cn/reference/directives-reference/#clientonly)禁用预渲染步骤。
+
+**解决方法**：尝试在渲染后访问这些对象（如 React 中的[`useEffect()`](https://reactjs.org/docs/hooks-reference.html#useeffect) 或 Vue 中的 [`onMounted()`](https://vuejs.org/api/composition-api-lifecycle.html#onmounted) 和 Svelte中的 [`onMount()`](https://svelte.dev/docs#run-time-svelte-onmount) ）。
+
+**状态**：预期 Astro 行为。
+
+#### 不是这个？
+
+**解决方案**：检查你的 [Astro](/zh-cn/core-concepts/astro-components/) 或 [UI 框架](/zh-cn/core-concepts/framework-components/)组件的相应文档。考虑从 [astro.new](https://astro.new) 中打开 Astro 入门模板，并在最小的 Astro 项目中对组件进行故障排除。
+
+**不确定这是你的问题？**
+检查一下是否有其他人报告过[这个问题](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Unable+to+render+Component)!
+
+
+### 预期的默认导出
+
+当试图导入或渲染无效组件，或无法正常工作的组件时，会出现这个错误。（该特定消息是因为 Astro 中导入 UI 组件方式造成的）
+
+**解决办法**：尝试寻找你正在导入和渲染的任何组件的错误，并确保它工作正常。考虑从 [astro.new](https://astro.new) 打开 Astro 入门模板，并在最小的 Astro 项目中对组件进行故障排除。
+
+**状态**：预期 Astro 行为。
+
+## 常见问题
+
+### 我的组件无法渲染
+
+首先，检查你是否在 [`.astro` 组件脚本](/zh-cn/core-concepts/astro-components/#组件-script)或 [`.md` frontmatter](/zh-cn/guides/markdown-content/#在-markdown-中使用组件) 中**导入了该组件**。
+
+然后检查你的导入语句。
+
+- 你的导入链接到了错误的地方吗？（检查你的导入路径）
+
+- 你的导入与导入的组件有名称相同吗？（检查你的组件名称和它是否[遵循 `.astro` 语法](/zh-cn/comparing-astro-vs-other-tools/#astro-vs-jsx)）
+
+- 你在导入时是否包含了扩展名？（检查你导入的文件是否包含扩展名。例如：`.astro`、`.md`、`.jsx`、`.vue`）
+
+### 我的组件没有互动
+
+如果你的组件正在渲染（见上文），但对用户的交互没有反应，那么你可能缺少 [`client:*`指令](/zh-cn/reference/directives-reference/#client-directives) 来对组件进行激活。
+
+默认情况下，[UI 框架组件在客户端不会激活](/zh-cn/core-concepts/framework-components/#激活组件) 。如果没有提供 `client:*` 指令，它的 HTML 将直接渲染到页面上，而无需 JavaScript。
+
+::tip
```

Comment:
Fix Aside syntax — details in the WRITING guide: https://github.com/withastro/docs/blob/main/WRITING.md#custom-aside-component

```suggestion
:::tip
```